typing/typemod.ml

```diff
@@ -3223,6 +3242,14 @@ let package_units initial_env objfiles cmifile modulename =
 
 open Printtyp
 
+let ref_manual_value_restriction () =
```

Comment:
Whenever possible, it is generally more forward compatible to define a printer rather than creating a string directly.

Reply:
Done, but this required changes elsewhere too (in other parts printing manual sections and in the check)

Reply:
Note that I didn't expect you to update all `(see manual section ...)` printer in the compiler while  adding a more general construct for the cross-reference checker, but the result does look nice.

Reply:
I am of two minds on the subject: on the one hand, the sets of interested reviewers for the two commits are probably the same so it is probably fine to group the first commit as a preliminary work for the second. On the other hand, the two commits are thematically distinct, and having separate PRs would help structure the history of changes in the compiler.
 
I will let you take the decision.